README.md

```diff
@@ -46,9 +55,13 @@ health-checker [options]
 | `--listener` |  The IP address and port on which inbound HTTP connections will be accepted. | `0.0.0.0:5000`
 | `--log-level` | Set the log level to LEVEL. Must be one of: `panic`, `fatal`, `error,` `warning`, `info`, or `debug` | `info` 
 | `--help` | Show the help screen | | 
+| `--script` | Path to script to run - will pass if it completes within configured timeout with a zero exit status. Specify one or more times. | | 
+| `--script-timeout` | Timeout, in seconds, to wait for the scripts to exit. Applies to all configured script targets. | `5` |  
 | `--version` | Show the program's version | | 
 
-#### Example
+If you execute a shell script, ensure you have a `shebang` line in your script, otherwise the script will fail with an `exec format error`.
```

Comment:
Could our code catch that? Seems like a simple `strings.HasPrefix("#!")` would do the trick to avoid some confusing errors?

Reply:
Catch here is that we can execute other targets than shell scripts. We would then first have to identify the script as shell script (via the extension or some other way) and only after that check the prefix within. I think it would unnecessarily clutter the code and potentially lead to even more confusing errors 😄